docs/usage/quickstart.rst

```diff
@@ -3,112 +3,112 @@
    Quickstart
    subdirectory
    Todo
-   usernames
 
 Quickstart Guide
 ================
-To quickly set up a testing cluster using MetalK8s_, you need 3 machines running
-CentOS_ 7.4 to which you have SSH access (these can be VMs). Each machine
-acting as a Kubernetes_ node (all of them, in this example) also need to have at
-least one disk available to provision storage volumes.
+To set up a testing cluster quickly using MetalK8s_ requires three machines
+running CentOS_ 7.4 to which you have SSH access (these can be VMs). Each
+machine acting as a Kubernetes_ node (all of them, in this example) must also
+have at least one disk available to provision storage volumes.
 
 .. todo:: Give some sizing examples
 
 .. _MetalK8s: https://github.com/scality/metal-k8s/
+
```

Comment:
Please don't add unnecessary whitespace etc. (happens a couple of times throughout the PR)

Reply:
Could you remove these empty lines (also other occurrences later in the file).